tasks/main.yml

```diff
@@ -1,5 +1,13 @@
 ---
 
+- name: Assert supported options
+  assert:
+    that:
+     - docker.thin_pool_device is none
+  when: ansible_distribution_release == "trusty"
+  tags:
+    - always
```

Comment:
always? That's new

Reply:
Yep. It works well in this case.

Reply:
How does it match to rest of the roles?

Reply:
Its not been used in the other roles. In this case, its a good solution as the assertion should be checked no matter what tags are provided.

Reply:
assert:
        that:
          - ansible_distribution_release == "trusty"

Would that work?

Reply:
no, because then the assertion would never be checked as no tags are specified.

Reply:
You don't need tag to trigger assert.
Assert is triggered automatically afaik

Reply:
could potentially specify tags of "build" and "configure" instead of "always"?

Reply:
I prefer not to create too many different/random tags.

Other wise LGTM

Reply:
With the tag in place: 
PLAY [Test Docker provision] ***************************************************

TASK [setup] *******************************************************************
ok: [default]

TASK [Update apt] **************************************************************
ok: [default]

TASK [sansible.docker : Assert supported options] ******************************
skipping: [default]

TASK [sansible.docker : Install essential packages] ****************************
ok: [default] => (item=[u'apt-transport-https', u'ca-certificates', u'curl'])

Reply:
w/o tag in place:
PLAY [Test Docker provision] ***************************************************

TASK [setup] *******************************************************************
ok: [default]

TASK [Update apt] **************************************************************
ok: [default]

TASK [sansible.docker : Install essential packages] ****************************
ok: [default] => (item=[u'apt-transport-https', u'ca-certificates', u'curl'])